frontend/lib/rehype-image-size.ts

```diff
@@ -9,7 +9,7 @@ import { Pluggable } from "unified";
 const rehypeImageSize: Pluggable = () => {
   return (tree) => {
     visit(tree, "element", (node) => {
-      if (node.tagName === "img") {
+      if (node.tagName === "img" && !node.properties.src.startsWith("http")) {
```

Comment:
どちらかというと絶対パスか否かをチェックしたほうがいいのではと思った

```suggestion
      if (node.tagName === "img" && node.properties.src.startsWith("/")) {
```